src/pages/all-about-carbon/why-use-carbon copy.mdx

```diff
@@ -0,0 +1,259 @@
+---
```

Comment:
is the `copy` at the end of the file name intentional? also does it need to be in `nav-items.yml`?

Reply:
No, it's not intentional. Let me fix. Thanks, Andrew!